__tests__/models/transaction.test.js

```diff
@@ -22,7 +22,7 @@ test('New tx', () => {
   const output1 = new Output(1000, address1);
   const output2 = new Output(1000, address2, {timelock: 1550249803});
   const inputDataHex = '4630440220317cd233801c1986c2de900bf8d344c6335d3c385e69d19d65e1fae7a0afd0af02207acddb824debf855798d79c45701cbe3a19aea00baad94bff5290c6f0b0acf8e210346cddff43dffab8e13398633ab7a7caf0d634551e89ae6fd563e282f6744b983';
-  const input1 = new Input('00034a15973117852c45520af9e4296c68adb9d39dc99a0342e23cd6686b295e', 0, {data: util.buffer.hexToBuffer(inputDataHex)})
+  const input1 = new Input('00034a15973117852c45520af9e4296c68adb9d39dc99a0342e23cd6686b295e', 0, {data: Buffer.from(inputDataHex, 'hex')})
```

Comment:
Can we use our `hexToBuffer` utils method here?

Reply:
We can, but since the input is a hard-coded hex, there is no difference between `hexToBuffer` and `Buffer.from`.